src/Morphir/TypeScript/Backend.elm

```diff
@@ -40,34 +47,95 @@ mapDistribution opt distro =
 
 mapPackageDefinition : Options -> Distribution -> Package.PackageName -> Package.Definition ta (Type ()) -> FileMap
 mapPackageDefinition opt distribution packagePath packageDef =
+    let
+        compilationUnitToFileMapElement : TS.CompilationUnit -> ( ( List String, String ), String )
```

Comment:
That `( ( List String, String ), String )` could be replaced with a type alias that describes what we actually want (perhaps `type alias FileMapElement = ( ( List String, String ), String )` ?).

Reply:
Good idea.